resources/patches/media_avportal-improve-exception-management.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/includes/media_avportal.formatters.inc b/includes/media_avportal.formatters.inc
+index 33638f8..181b66d 100644
+--- a/includes/media_avportal.formatters.inc
++++ b/includes/media_avportal.formatters.inc
+@@ -53,9 +53,14 @@ function media_avportal_file_formatter_video_view($file, $display) {
+     // Let's add the language to the options.
+     global $language;
+     $wrapper = file_stream_wrapper_get_instance_by_uri($file->uri);
+-    $languages = $wrapper->getAvailableVideoLanguage();
++    try {
++      $languages = $wrapper->getAvailableVideoLanguage();
++    }
++    catch (Exception  $e) {
++      watchdog_exception('media_avportal', $e);
++    }
+     $video_language = 'en';
+-    if (in_array($language->language, $languages)) {
++    if (is_array($languages) && in_array($language->language, $languages)) {
```

Comment:
This throws an error when the ws is unavailable. The variable $languages is not set.